.github/workflows/orchestrate-bootstrap.yaml

```diff
@@ -0,0 +1,34 @@
+name: Orchestrate bootstrapper workflow
```

Comment:
is that a zero or a capital o ?

Reply:
A capital o(O) (in my VSC code at least a 0 has a line through it to denote the difference)